module4/backend/modules/database.js

```diff
@@ -26,7 +26,7 @@ mongoose.connection.once('open', logConnectionResult);
 
 const schema = {
   name: String,
-  cost: Number,
+  cost: String,
```

Comment:
any reason why changed from number to string?

Reply:
As there is no computation taking place that would benefit from a number, this solves a bug that was causing values such as "100.00" to render on the frontend as "100" and numbers like "322.20" to render as "322.2"